web/src/components/FilingsCalendarTaxAccess.tsx

```diff
@@ -0,0 +1,74 @@
+import { Content } from "@/components/Content";
+import { SnackbarAlert } from "@/components/njwds-extended/SnackbarAlert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { LookupLegalStructureById, LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/";
+
+import { ReactElement, useState } from "react";
+import { Gov2GoModal } from "./Gov2GoModal";
```

Comment:
same comment as elsewhere about dependency inversion. Especially on the frontend, we should never be naming a component after a specific backend API implementation that we don't have control over. Instead, this should be named in a way that describes its purpose in the context of the user and the frontend application experience